test/errorReports/too_long_line.sol.log

```diff
@@ -0,0 +1,7 @@
+Warning: This is a pre-release compiler version, please do not use it in production.
```

Comment:
You might have to find a way to filter this out - it will not be there for a release version and thus the tests would fail for the release version...

Reply:
sure, thx.